analysis/priestholy/src/modules/shadowlands/items/FlashConcentration.tsx

```diff
@@ -0,0 +1,228 @@
+  }
+
+  /**
+   * Sets _isFullStack to false.
+   * Note: The only exception is the prepull event, for which we assume that
+   * it means the character has five stacks. It will be corrected if the
+   * first Flash Heal generates a ApplyBuffStack or ApplyBuff event
+   * Note 2: Technically, the only time this function does something is on the first
+   * apply buff if there was no prepull FC.
+   * @param event with info about first stack, or prepull incomplete indication
+   */
+  onByPlayerFlashConcentrationInitialBuff(event: ApplyBuffEvent) {
+    if (event.prepull !== true) {
+      this._isFullStack = false;
+      this._currentStacks = 1;
+    }
+  }
+
+  /**
+   * Increments _totalDrops for our suggestion, since that should ideally not happen
+   */
+  onByPlayerFlashConcentrationBuffRemoval(event: RemoveBuffEvent) {
+    this._totalDrops += 1;
+    this._isFullStack = false;
+    this._currentStacks = 0;
+  }
+
+  /**
+   * when at full stack, it refreshes the last refresh timer,
+   * and counts a Wasteful Cast if it was used earlier than needed
+   */
+  onByPlayerFlashHealCast(event: CastEvent) {
+    if (this._isFullStack) {
+      if (event.timestamp - this._lastFullStackRefresh < 15000) {
+        this._wastefulCasts += 1;
+      }
+      this._lastFullStackRefresh = event.timestamp;
+    }
+  }
+
+  onByPlayerHealAmount(event: HealEvent) {
+    this._totalHeal += event.amount;
+    this._gainedHeal +=
+      (event.amount * (this._currentStacks * 0.03)) / (1 + this._currentStacks * 0.03);
+    this._gainedCastTime += this._currentStacks * 200;
+  }
+
+  onByPlayerHealCast(event: CastEvent) {
+    this._totalCastTime +=
+      event.timestamp -
+      (event.channel?.beginChannel?.timestamp
+        ? event.channel.beginChannel.timestamp
+        : event.timestamp);
+  }
+
+  statistic() {
+    // HealHPSNoFC gives the HPS which would have been obtained by casting the same number of heals without Flash Concentration Buff
+    const HealHPSNoFC =
+      (this._totalHeal - this._gainedHeal) / (this._totalCastTime + this._gainedCastTime);
+    // BuffedHealHPS gives the total HPS of Heal in this fight
+    const BuffedHealHPS = this._totalHeal / this._totalCastTime;
+    return (
+      <Statistic
+        size="flexible"
+        category={STATISTIC_CATEGORY.ITEMS}
+        tooltip={
+          <>
+            Impact is approximative since <SpellLink id={SPELLS.FLASH_CONCENTRATION.id} /> modifies
+            the gameplay in depth
+          </>
+        }
+      >
+        <BoringSpellValueText spell={SPELLS.FLASH_CONCENTRATION}>
+          <ItemHealingDone
+            // approximate considering the player would have cast heal without FC
+            amount={(BuffedHealHPS - HealHPSNoFC) * this._totalCastTime}
+            approximate
+          />
+          <br />
+          {this._gainedCastTime / 1000} sec saved on <SpellLink id={SPELLS.GREATER_HEAL.id} /> casts
+        </BoringSpellValueText>
```

Comment:
What does this look like? Can you attach a screenshot?

Reply:
Here goes :)
![image](https://user-images.githubusercontent.com/9220340/119743846-5ce36680-be50-11eb-9a0a-5e62720a0a90.png)

Reply:
Having `40.6 sec saved on Heal casts` split over two lines doesn't look great. You could fit it on one line using `<small>` tags to produce the muted text like you see with `11.82% of total`